admin/data/sql/migrations/20190210204450.do.grant-delete-school-score.js

```diff
@@ -0,0 +1,5 @@
+const config = require('../../../config')
+
+module.exports.generateSql = () => {
+  return `GRANT DELETE ON [mtc_admin].[schoolScore] TO ${config.Sql.Application.Username}`
```

Comment:
you could just use the default of `UPDATE` and set the column to NULL.  this would be the least permissive option, which is always preferable.

Reply:
@GuyHarwood if I use `UPDATE` instead then I need to add more calls to determine if I need to perform insert (on the very first data submission) or update for the school score dataset.

Reply:
As SQL Server does not quite have Upsert; some patterns are listed here:
http://michaeljswart.com/2017/07/sql-server-upsert-patterns-and-antipatterns/

Reply:
@GuyHarwood @jon-shipley resolving this as the comments concern the older implementation